criu/clone-noasan.c

```diff
@@ -80,5 +80,7 @@ int clone3_with_pid_noasan(int (*fn)(void *), void *arg, int flags,
 	pid = syscall(__NR_clone3, &c_args, sizeof(c_args));
 	if (pid == 0)
 		exit(fn(arg));
+	if (pid < 0)
+		return -errno;
```

Comment:
This function can return -1, but this isn't -EPERM. Do we really need to return -errno here? Maybe it is better to write a comment that this functions returns valid errno in cases of errors and initialize it properly where it returns -1.

Reply:
How do I initialize it properly?

Reply:
if (exit_signal != SIGCHLD) {
		pr_err("Exit signal not SIGCHLD\n");
                errno = EINVAL;
		return -1;
}

Reply:
I changed the code like you suggested.